RecoHI/HiJetAlgos/plugins/HiBadParticleCleaner.cc

```diff
@@ -0,0 +1,245 @@
+// system include files
+#include <memory>
+#include <iostream>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+#include "DataFormats/MuonReco/interface/MuonSelectors.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidateFwd.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+//
+// class declaration
+//
+
+class HiBadParticleCleaner : public edm::global::EDProducer<> {
+public:
+  explicit HiBadParticleCleaner(const edm::ParameterSet&);
+  ~HiBadParticleCleaner() override;
```

Comment:
Override with the default destructor

Reply:
Sorry, what is the suggestion exactly?

Reply:
```suggestion
  ~HiBadParticleCleaner() override = default;
```
(since your destructor is expected doing nothing, better to stick on the default one)